recipes/dorado/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "dorado" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://cdn.oxfordnanoportal.com/software/analysis/dorado-{{ version }}-linux-x64.tar.gz  # [linux and x86_64]
```

Comment:
This downloads 2.0 GB ! I guess the package includes some data.
```
❯ ll dorado-0.7.1-linux-x64/bin/
.rwxr-xr-x martin martin 1.8 GB Mon Jun  3 20:19:21 2024  dorado
```

Would it be better to build from sources ? I.e. https://github.com/nanoporetech/dorado/archive/refs/tags/v0.7.1.tar.gz